build/ci/phase-template.yml

```diff
@@ -14,7 +14,10 @@ phases:
       _arch: ${{ parameters.architecture }}
       _codeCoverage: ${{ parameters.codeCoverage }}
     queue:
-      timeoutInMinutes: 45
+      ${{ if eq(variables._codeCoverage, 'false') }}:
+        timeoutInMinutes: 45
+      ${{ if eq(variables._codeCoverage, 'true') }}:
+        timeoutInMinutes: 90
```

Comment:
I think 90 min might be too much if the builds take around 45 min. We unfortunately still have some builds that get stuck and the build completion time does not oscillate that much when they terminate. I would increase it to 60 min as Ivan suggested.

Reply:
Yes, I scanned a bunch of builds and the avg time is 42 mins. Let's start with 60 mins and see how it goes.

Reply:
Done